services/galley/src/Galley/API/Federation.hs

```diff
@@ -56,12 +55,12 @@ updateConversationMemberships cmu = do
   when (not (null localUsers)) $ do
     Data.addLocalMembersToRemoteConv localUserIds (cmuConvId cmu)
   -- FUTUREWORK: the resulting event should have qualified users and conversations
```

Comment:
that futurework is done I suppose (feel free to remove this comment only in a follow-up PR though so you don't need to wait for CI again)